src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4155,6 +4398,23 @@ namespace AirflowNetworkBalanceManager {
 		static bool OneTimeFlag( true );
 		static bool ErrorsFound( false );
 		Real64 GlobalOpenFactor;
+		Real64 ZonePressure1;
```

Comment:
Seems you should initialize all of these here or move some to be declared within a tighter scope